docs/modules/vector-search/pages/vector-collections.adoc

```diff
@@ -0,0 +1,498 @@
+= Vector Collection
+
+== Intro
+
+The primary object for interacting with the database is a Vector Collection. This collection holds information about the vectors and metadata (value).
+
+A collection can consist of one or more indexes, all sharing a single storage metadata. Each index represents a distinct vector space. Vectors from different indexes are stored independently, and indexes can have different configurations.
+
+NOTE: At this point, each metadata object must correspond to exactly one vector in each index.
+
+NOTE: When working with multiple indexes in your collections, it's necessary to specify the index to search in your queries..
```

Comment:
this note is too specific for intro, it should be in section about search

Reply:
note deleted:
97e456281191372558f94773bdb743a9c1f92e4f